### PR TITLE
Another try to fix DruidCoordinatorTest#testCoordinatorRun

### PR DESCRIPTION
**Pull request by navis**
Yet another try (sorry).

### CODE REVIEW

**Comment by navis** [comment]
Passed once.

File: server/src/test/java/io/druid/server/coordinator/DruidCoordinatorTest.java [hunk 1]
```diff
@@ -342,6 +342,9 @@ public void childEvent(
       Thread.sleep(50);
     }
 
+    // wait historical data to be updated
+    Thread.sleep(druidCoordinatorConfig.getCoordinatorPeriod().getMillis());
```
Comment (fjy) [inline server/src/test/java/io/druid/server/coordinator/DruidCoordinatorTest.java]: how long is the default here? we dont want unit tests that take minutes to complete

Comment (navis) [inline server/src/test/java/io/druid/server/coordinator/DruidCoordinatorTest.java]: `COORDINATOR_PERIOD` which is 100 msec in this test.

Comment (navis) [inline server/src/test/java/io/druid/server/coordinator/DruidCoordinatorTest.java]: .. or do you want wait on update with lock?

Comment (fjy) [inline server/src/test/java/io/druid/server/coordinator/DruidCoordinatorTest.java]: this period is fine

Comment (fjy) [inline server/src/test/java/io/druid/server/coordinator/DruidCoordinatorTest.java]: but I think we'll still see sporadic failures unless there's more of a check in place. Can we add some sort of conditional check?

**Comment by fjy** [comment]
:+1:

**Pull request merged**